2025-02-28 22:39:39,545 INFO    MainThread:29360 [wandb_init.py:setup_run_log_directory():637] Logging user logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250228_223939-exuju8p4\logs\debug.log
2025-02-28 22:39:39,545 INFO    MainThread:29360 [wandb_init.py:setup_run_log_directory():638] Logging internal logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250228_223939-exuju8p4\logs\debug-internal.log
2025-02-28 22:39:39,546 INFO    MainThread:29360 [wandb_init.py:monkeypatch_ipython():589] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001F8D36B43A0>
2025-02-28 22:39:39,552 INFO    MainThread:29360 [wandb_init.py:init():756] calling init triggers
2025-02-28 22:39:39,553 INFO    MainThread:29360 [wandb_init.py:init():761] wandb.init called with sweep_config: {}
config: {'epochs': 500, 'activation': 'relu', 'nb_layers': 3, 'nb_neurons': 128, 'dropout_rate': 0.1, 'optimizer': 'adam', 'loss': 'mae', 'metric': 'mae', 'augmentation': True, 'image_resize': 256, '_wandb': {}}
2025-02-28 22:39:39,553 INFO    MainThread:29360 [wandb_init.py:init():778] found existing run on stack: v57ccdjh
2025-02-28 22:39:39,553 INFO    MainThread:29360 [wandb_run.py:_finish():2110] finishing run cyrilfrebel-none/eyeball tracker/v57ccdjh
2025-02-28 22:39:39,554 INFO    MainThread:29360 [wandb_run.py:_atexit_cleanup():2350] got exitcode: 0
2025-02-28 22:39:39,554 INFO    MainThread:29360 [wandb_run.py:_restore():2332] restore
2025-02-28 22:39:39,555 INFO    MainThread:29360 [wandb_run.py:_restore():2338] restore done
2025-02-28 22:39:41,769 INFO    MainThread:29360 [wandb_run.py:_footer_history_summary_info():3963] rendering history
2025-02-28 22:39:41,770 INFO    MainThread:29360 [wandb_run.py:_footer_history_summary_info():3995] rendering summary
2025-02-28 22:39:41,771 INFO    MainThread:29360 [wandb_run.py:_footer_sync_info():3924] logging synced files
2025-02-28 22:39:41,777 INFO    MainThread:29360 [wandb_init.py:init():789] starting backend
2025-02-28 22:39:41,777 INFO    MainThread:29360 [wandb_init.py:init():793] sending inform_init request
2025-02-28 22:39:41,785 INFO    MainThread:29360 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=spawn, using: spawn
2025-02-28 22:39:41,840 INFO    MainThread:29360 [wandb_init.py:init():808] backend started and connected
2025-02-28 22:39:41,856 INFO    MainThread:29360 [wandb_run.py:_label_probe_notebook():1196] probe notebook
2025-02-28 22:39:41,858 INFO    MainThread:29360 [wandb_init.py:init():901] updated telemetry
2025-02-28 22:39:41,861 INFO    MainThread:29360 [wandb_init.py:init():936] communicating run to backend with 90.0 second timeout
2025-02-28 22:39:42,232 INFO    MainThread:29360 [wandb_init.py:init():994] starting run threads in backend
2025-02-28 22:39:42,583 INFO    MainThread:29360 [wandb_run.py:_console_start():2385] atexit reg
2025-02-28 22:39:42,583 INFO    MainThread:29360 [wandb_run.py:_redirect():2235] redirect: wrap_raw
2025-02-28 22:39:42,583 INFO    MainThread:29360 [wandb_run.py:_redirect():2300] Wrapping output streams.
2025-02-28 22:39:42,583 INFO    MainThread:29360 [wandb_run.py:_redirect():2325] Redirects installed.
2025-02-28 22:39:42,585 INFO    MainThread:29360 [wandb_init.py:init():1036] run started, returning control to user process
2025-02-28 22:39:53,093 INFO    MainThread:29360 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-28 22:39:53,094 INFO    MainThread:29360 [wandb_init.py:_pause_backend():554] pausing backend
2025-02-28 22:40:17,131 INFO    MainThread:29360 [wandb_init.py:_resume_backend():559] resuming backend
2025-02-28 22:40:17,138 INFO    MainThread:29360 [wandb_init.py:setup_run_log_directory():637] Logging user logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250228_224017-smdov1zc\logs\debug.log
2025-02-28 22:40:17,139 INFO    MainThread:29360 [wandb_init.py:setup_run_log_directory():638] Logging internal logs to c:\Users\Cyril\Desktop\data-science-lab\eyeball tracker\wandb\run-20250228_224017-smdov1zc\logs\debug-internal.log
2025-02-28 22:40:17,139 INFO    MainThread:29360 [wandb_init.py:init():756] calling init triggers
2025-02-28 22:40:17,139 INFO    MainThread:29360 [wandb_init.py:init():761] wandb.init called with sweep_config: {}
config: {'epochs': 500, 'activation': 'relu', 'nb_layers': 3, 'nb_neurons': 128, 'dropout_rate': 0.1, 'optimizer': 'adam', 'loss': 'mae', 'metric': 'mae', 'augmentation': True, 'image_resize': 256, '_wandb': {}}
2025-02-28 22:40:17,139 INFO    MainThread:29360 [wandb_init.py:init():778] found existing run on stack: exuju8p4
2025-02-28 22:40:17,139 INFO    MainThread:29360 [wandb_run.py:_finish():2110] finishing run cyrilfrebel-none/eyeball tracker/exuju8p4
2025-02-28 22:40:17,141 INFO    MainThread:29360 [jupyter.py:save_history():458] not saving jupyter history
2025-02-28 22:40:17,141 INFO    MainThread:29360 [jupyter.py:save_ipynb():386] not saving jupyter notebook
2025-02-28 22:40:17,141 INFO    MainThread:29360 [wandb_init.py:_jupyter_teardown():571] cleaning up jupyter logic
2025-02-28 22:40:17,141 INFO    MainThread:29360 [wandb_run.py:_atexit_cleanup():2350] got exitcode: 0
2025-02-28 22:40:17,141 INFO    MainThread:29360 [wandb_run.py:_restore():2332] restore
2025-02-28 22:40:17,141 INFO    MainThread:29360 [wandb_run.py:_restore():2338] restore done
